backend/core/src/listings/listings.service.spec.ts

```diff
@@ -194,6 +196,56 @@ describe("ListingsService", () => {
       )
     })
 
+    it("should support basic filters with null data", async () => {
+      mockListingsRepo.createQueryBuilder
+        .mockReturnValueOnce(mockInnerQueryBuilder)
+        .mockReturnValueOnce(mockQueryBuilder)
+      const queryParams: ListingsQueryParams = {
+        filter: [
+          {
+            $comparison: Compare["="],
+            name: "minRent",
+            $include_nulls: true,
+          },
+        ],
+      }
+
+      const listings = await service.list(queryParams)
+
+      expect(listings.items).toEqual(mockListings)
+      expect(mockInnerQueryBuilder.andWhere).toHaveBeenCalledWith(
+        "(LOWER(CAST(listings.name as text)) = LOWER(:name_0) OR listings.name IS NULL)",
+        {
+          name_0: "minRent",
+        }
+      )
+    })
+
+    it("should support custom filters with null data", async () => {
```

Comment:
similar nit here: can we name this slightly more descriptively so it answers: What does "support" mean in this context?